tests/test_x509.py

```diff
@@ -3751,23 +3792,26 @@ def test_init_bad_value(self):
                 b'bytes'
             )
 
-    def test_init_bad_country_code_value(self):
-        with pytest.raises(ValueError):
-            x509.NameAttribute(
+    def test_bad_country_code_value(self):
```

Comment:
Should we keep these tests? This same logic is tested in `test_calls_validator` as well.